lib/mpl_toolkits/axes_grid1/anchored_artists.py

```diff
@@ -232,7 +232,7 @@ class AnchoredSizeBar(AnchoredOffsetbox):
     def __init__(self, transform, size, label, loc,
                  pad=0.1, borderpad=0.1, sep=2,
                  frameon=True, size_vertical=0, color='black',
-                 label_top=False, fontproperties=None,
+                 fill=False, label_top=False, fontproperties=None,
```

Comment:
please put this last to not break the API.  Technically someone could be passing in 12 positional arguments.